extensions/ruby/src/language_servers/rubocop.rs

```diff
@@ -0,0 +1,19 @@
+use zed_extension_api::{self as zed, Result};
+
+pub struct Rubocop {}
+
+impl Rubocop {
+    pub const LANGUAGE_SERVER_ID: &'static str = "rubocop";
+
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    pub fn server_script_path(&mut self, worktree: &zed::Worktree) -> Result<String> {
+        let path = worktree.which("rubocop").ok_or_else(|| {
```

Comment:
I just realised that all language servers in Ruby extension do not respect the `binary` settings that Zed provides to set the binary path. I will address that in a follow-up pull request.